src/Core/HtmlResponse.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace XdebugDotOrg\Core;
+
+/**
+ * @psalm-immutable
```

Comment:
Could you explain how this whole psalm thing works? And I did see correctly that I don't really have to deploy that as it's `require-dev` ?

Reply:
This is for later – I haven't added a Psalm config yet - but it'll work similarly to this: https://github.com/psalm/psalm.dev/blob/master/psalm.xml

The idea is that the more complicated a site gets, the more important it is to make sure that you're not breaking things. Psalm exists to prevent errors, and this immutable annotation is [an extra layer on top](https://psalm.dev/articles/immutability-and-beyond).

Reply:
Once I have it configured, you'll run `vendor/bin/psalm` and hopefully see no errors.

Reply:
Things like this: https://github.com/xdebug/xdebug.org/pull/66/commits/f1034eb673ad002603c2eae8b7d87852f8cc9630